src/sonic-yang-models/tests/yang_model_tests/test_yang_model.py

```diff
@@ -16,7 +16,7 @@
 FAIL = 1
 logging.basicConfig(level=logging.DEBUG)
 log = logging.getLogger("YANG-TEST")
-log.setLevel(logging.INFO)
+log.setLevel(logging.WARNING)
```

Comment:
why do you change log level?

Reply:
I found that when running the unit tests, a lot of info level logs would be printed for tests without issues which made it difficult to find which tests were actually failing.